docs/theme-specification.md

```diff
@@ -131,68 +160,55 @@ m: [ 0, 1, 2 ]
 }
 ```
 
-#### Media Queries
-
-For convenience and for use with other styling approaches, a `mediaQueries` scale derived from the `breakpoints` scale can be added to the theme object.
```

Comment:
Why ditch this? I prefer min-width media queries, but do see the value in allowing other media queries, even `@media print`

Reply:
sorry I went I see small, medium large xxl etc I short circuit and reject the idea :D mostly because there will always be that one-off where maybe you are going to need `aBitBiggerThanSmallButNotMedium`. Also I think that the future is about element queries / resize observer / conditional rendering based on local context rather than global (the entire viewport).

That said the `mediaQueries` section can be useful (and many will keep using media queries) so I'd bring it back. I wonder if there is a better way to classify them that is as idiomatic as `small`,`medium`.

Reply:
Yeah, makes sense. Using arrays still leaves room for that "in-between" size issue. I mostly used `small`, `medium`, `large` to convey they idea of what `mediaQueries` was for, but it's certainly not the best naming convention

Reply:
I think a theme specification should include some notion of breakpoints since it's intimately related to scales, colors, etc. when building/designing. IMO if a collection of media queries can handle 90%+ of the usage then it's a success. There will always be the need for one off mqs where something needs to be in between two values. That's something we don't necessarily need to accommodate.